### PR TITLE
Added an example for the manim colormap

### PR DESCRIPTION
**Pull request by kolibril13**
## Changelog / Overview
Added an example for the manim colormap

## Checklist
- [ ] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
- [ ] I have written a descriptive PR title (see top of PR template for examples)
- [ ] I have written a changelog entry for the PR or deem it unnecessary
- [ ] My new functions/classes either have a docstring or are private
- [ ] My new functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] My new documentation builds, looks correctly formatted, and adds no additional build warnings

## Reviewer Checklist
- [ ] The PR title is descriptive enough
- [ ] The PR is labeled correctly
- [ ] The changelog entry is completed if necessary
- [ ] Newly added functions/classes either have a docstring or are private
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings

### CODE REVIEW

**Comment by kolibril13** [comment]
see here https://manimce--1242.org.readthedocs.build/en/1242/reference/manim.utils.color.Colors.html#colorexample

File: manim/utils/color.py [hunk 1]
```diff
@@ -105,6 +105,19 @@ class Colors(Enum):
...
+        :save_last_frame:
+
+        from manim.utils.color import Colors
+        class ColorExample(Scene):
+            def construct(self):
+                cols= Colors._member_names_
+                s= VGroup(*[Line(DOWN, UP, stroke_width=15).set_color(Colors[cols[i]].value) for i in range(0,len(cols))])
+                s.arrange_submobjects(buff=0.2)
+                self.add(s)
+                self.wait()
```
Comment (naveen521kk) [inline manim/utils/color.py:119]: seem's useless
```suggestion
```

Comment (kolibril13) [reply manim/utils/color.py:119]: thanks, that was an artefact.

File: manim/utils/color.py [hunk 2]
```diff
@@ -105,6 +105,18 @@ class Colors(Enum):
...
     --------
+
+    .. manim:: ColorExample
+        :save_last_frame:
+
+        from manim.utils.color import Colors
+        class ColorExample(Scene):
+            def construct(self):
+                cols= Colors._member_names_
+                s= VGroup(*[Line(DOWN, UP, stroke_width=15).set_color(Colors[cols[i]].value) for i in range(0,len(cols))])
```
Comment (naveen521kk) [inline manim/utils/color.py:116]:
```suggestion
                cols = Colors._member_names_
                s = VGroup(*[Line(DOWN, UP, stroke_width=15).set_color(Colors[cols[i]].value) for i in range(0, len(cols))])
```

**Review by naveen521kk — APPROVED** [review]
sgtm

**Review by jsonvillanueva — APPROVED** [review]

**Pull request merged**